Envoy.blade.php

```diff
@@ -210,6 +213,34 @@
     fi
 @endtask
 
+@task('deploy:cronjobs')
+    @if (is_array($cronJobs))
+        @if (count($cronJobs) > 0)
+            FILE=$(mktemp)
+
+            echo '# WARNING! THIS FILE IS MANAGED BY ENVOY. DO NOT EDIT!' >> $FILE
+            echo 'SHELL="/bin/bash"' >> $FILE
+            echo 'MAILTO="{{ $cronMailTo }}"' >> $FILE
+            echo >> $FILE
+
+            @foreach ($cronJobs as $cronJob)
+                echo {{ escapeshellarg($cronJob) }} >> $FILE
+            @endforeach
+
+            crontab $FILE
+            rm $FILE
+        @else
+            crontab -r || true
```

Comment:
We should check that the `crontab` is "MANAGED BY ENVOY" before deleting it. Otherwise, we could lose the `crontab` that a user might have defined.

Reply:
Fixed above.